CONTRIBUTING.md

```diff
@@ -1,16 +1,30 @@
 # mongodb-operator
 
-## Code overview
 
-TODO: expand on this section when spec doc is created
+## Developing
 
-## Intended use case
+As this is a machine charm, it can be deployed locally using [Juju on a local LXD cloud](https://juju.is/docs/olm/lxd). Once the controller is created, to deploy this workload you need to follow some preliminar steps.
 
-TODO: expand on this section when spec doc is created
+Afer downloading the repository, create and activate a virtualenv, and install the development requirements:
 
-## Roadmap
+```bash
+$ virtualenv -p python3 venv
+$ source venv/bin/activate
+$ pip install -r requirements.txt
+```
+
+### Build
+
+```bash
+$ charmcraft pack
+```
+
+### Deploy
+
+```bash
+$ juju deploy ./mongodb_ubuntu-20.04-amd64.charm
```

Comment:
remove `$` it makes it difficult for people to copy and paste into the terminal :-)